src/main/windows/mainWindow.ts

```diff
@@ -65,6 +65,16 @@ function createMainWindow(options: {linuxAppIcon: string}) {
     const {maximized: windowIsMaximized} = savedWindowState;
 
     const spellcheck = (typeof Config.useSpellChecker === 'undefined' ? true : Config.useSpellChecker);
+    const isFullScreen = (savedWindowStateFullscreen: boolean | undefined) => {
+        if (global?.args?.fullscreen !== undefined) {
+            return global.args.fullscreen;
+        }
+
+        if (Config.startInFullscreen) {
+            return Config.startInFullscreen;
+        }
+        return savedWindowStateFullscreen ?? undefined;
+    };
```

Comment:
I don't think you need any parameters for this function as they are not going to change from the definition.

```suggestion
    const isFullScreen = () => {
        if (global?.args?.fullscreen !== undefined) {
            return global.args.fullscreen;
        }

        if (Config.startInFullscreen) {
            return Config.startInFullscreen;
        }
        return options.fullscreen || savedWindowState.fullscreen || false
    };
```